shard_to_single_vcf.py

```diff
@@ -0,0 +1,65 @@
+## THIS SCRIPT NEEDS SSDs!!!!
+import argparse
+
+import hail as hl
+from gnomad.utils.slack import try_slack
+
+
+def main(args):
+
+    hl.init(log="/vcf_write.log", default_reference=f"{args.build}")
+
+    mt = hl.import_vcf(
+        f"{args.genotypes_vcf}", force_bgz=True, find_replace=("nul", ".")
+    )
+    filters_ht = hl.import_vcf(
+        f"{args.filtered_vcf}", force_bgz=True, find_replace=("nul", ".")
+    ).rows()
+    mt = mt.annotate_rows(filters=filters_ht[mt.row_key].filters)
+    hl.export_vcf(
+        mt,
+        f"gs://seqr-datasets/v02/{args.build}/RDG_{args.data_type}_Broad_{args.data_source}/v{args.version}/RDG_{args.data_type}_Broad_{args.data_source}.vcf.bgz",
```

Comment:
so I learned that this is pretty slow by running it on the exome callset, is there any reason this can't be sharded?

Reply:
That's a good point, I'll have to check out the loading pipeline code to make sure it won't get angry if it finds a sharded vcf.

Reply:
Actually, now that I think of it, hail has that strange bug where you need vcf.bgz for it to block zip the shards but in doing so the vcf.bgz becomes a directory and the pipeline won't match with that. So I think we could do a post processing gsutil mv but with googles storage library within this script?

Reply:
oh cool, I didn't know you could do that. do you think that would work? something like this? https://stackoverflow.com/questions/25972482/how-to-move-files-in-google-cloud-storage-from-one-bucket-to-another-bucket-by-p